app/src/main/java/at/roteskreuz/stopcorona/model/repositories/DashboardRepository.kt

```diff
@@ -2,48 +2,26 @@ package at.roteskreuz.stopcorona.model.repositories
 
 import android.content.SharedPreferences
 import at.roteskreuz.stopcorona.constants.Constants
-import at.roteskreuz.stopcorona.model.db.dao.NearbyRecordDao
 import at.roteskreuz.stopcorona.skeleton.core.utils.booleanSharedPreferencesProperty
-import at.roteskreuz.stopcorona.utils.asDbObservable
-import io.reactivex.Observable
 
 /**
  * Repository for managing dashboard content.
  */
 interface DashboardRepository {
 
-    val showMicrophoneExplanationDialog: Boolean
-
-    /**
-     * Observes the number of met people.
-     */
-    fun observeSavedEncountersNumber(): Observable<Int>
-
     /**
-     * Do not show explanation dialog again.
+     * Information if the automatic handshake was enabled automatically on the first start
      */
-    fun setMicrophoneExplanationDialogShown()
+    var serviceEnabledOnFirstStart: Boolean
 }
 
 class DashboardRepositoryImpl(
-    private val nearbyRecordDao: NearbyRecordDao,
     preferences: SharedPreferences
 ) : DashboardRepository {
 
     companion object {
-        private const val PREF_MICROPHONE_EXPLANATION_DIALOG_SHOW_AGAIN =
-            Constants.Prefs.DASHBOARD_PREFIX + "microphone_explanation_dialog_show_again"
+        private const val PREF_SERVICE_ENABLED_ON_FIRST_START = Constants.Prefs.DASHBOARD_PREFIX + "service_enabled_on_first_start"
     }
 
-    override var showMicrophoneExplanationDialog: Boolean
-        by preferences.booleanSharedPreferencesProperty(PREF_MICROPHONE_EXPLANATION_DIALOG_SHOW_AGAIN, true)
-        private set
-
-    override fun observeSavedEncountersNumber(): Observable<Int> {
-        return nearbyRecordDao.observeNumberOfRecords().asDbObservable()
-    }
-
-    override fun setMicrophoneExplanationDialogShown() {
-        showMicrophoneExplanationDialog = false
-    }
+    override var serviceEnabledOnFirstStart: Boolean by preferences.booleanSharedPreferencesProperty(PREF_SERVICE_ENABLED_ON_FIRST_START, false)
```

Comment:
this should all be removed IMHO

Reply:
not removed, renamed. https://github.com/austrianredcross/stopp-corona-android/pull/83#discussion_r433731286

Reply:
yes - renamed!